### PR TITLE
Burger Menu

### PR DESCRIPTION
**Pull request by JelaniThompson**
image

image

image

A ton of help from @alex-wilmer and @mihok behind the scenes. Thanks guys 👍

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,8 +31,7 @@
     "commit": "gitmoji -c",
     "deploy": "npm run build&&gh-pages -d build",
     "eject": "react-scripts eject",
-    "precommit": "eslint src --fix",
```
Comment (alex-wilmer) [inline package.json]: did you mean to remove this?

Comment (JelaniThompson) [inline package.json]: @alex-wilmer Fixed! Sorry, I pasted in an older version by mistake.

**Comment by alex-wilmer** [comment]
hmm looks like the precommit didn't work. did you run yarn again when syncing with master?

File: package.json [hunk 2]
```diff
@@ -11,13 +11,13 @@
     "flow-bin": "^0.44.2",
     "gh-pages": "^0.12.0",
     "gitmoji-cli": "^1.5.2",
-    "husky": "^0.13.3",
```
Comment (alex-wilmer) [inline package.json]: this should be put back as well

Comment (JelaniThompson) [inline package.json]: Hey, I did a `git pull` and a `yarn upgrade` right after. Hopefully the pre-commit triggers correctly.

Comment (JelaniThompson) [inline package.json]: Okay I don't think it triggered correctly but now I'm getting the error of `Cannot find module 'eslint-config-minimal'` even though I have it installed in the project. This is after I tried manually running the pre-commit command `eslint src --fix`.

Minimal is referenced in my `.eslintrc`.

Comment (alex-wilmer) [inline package.json]: definitely something wonky happening. you can see eslint-config-minimal in your node_modules folder? maybe try killing it and running yarn again

**Comment by JelaniThompson** [comment]
Okay so I navigated to `./node_modules/.bin` and tried executing the pre-commit command from there. No files were changed so I think it might've worked this time around. Please let me know if I'm wrong @alex-wilmer. 👍

**Comment by kieran** [comment]
closing this PR as this is woefully out of date, but happy to consider a new nav PR!

image

**Pull request closed without merging**